broker/utils.go

```diff
@@ -10,19 +10,18 @@ import (
 	composeapi "github.com/compose/gocomposeapi"
 )
 
+var errDeploymentNotFound = errors.New("Deployment not found")
+
 func findDeployment(c compose.Client, name string) (*composeapi.Deployment, error) {
```

Comment:
We discussed this offline and agreed to keep it as it is and try to do upstream PR in the future to improve error handling.